theia-extensions/viewer-prototype/src/browser/trace-explorer/trace-explorer-sub-widgets/trace-explorer-placeholder-widget.tsx

```diff
@@ -40,16 +40,16 @@ export class TraceExplorerPlaceholderWidget extends ReactWidget {
     @postConstruct()
     init(): void {
         this.preferenceService.onPreferenceChanged(event => {
-            if (event.preferenceName === TRACE_PORT) {
-                this.port = event.newValue;
+            if (event.preferenceName === TRACE_ARGS) {
+                this.args = event.newValue;
             }
             if (event.preferenceName === TRACE_PATH) {
                 this.path = event.newValue;
             }
         });
```

Comment:
Port change events should still be handled, no?

Reply:
We replaced the port variable with args in this file, hence Port change events wouldn't affect any implementation